mikeio1d/res1d.py

```diff
@@ -136,6 +138,11 @@ def _add_node(self, node_id):
     def _add_catchment(self, catchment_id):
         self._data_subfilter.Catchments.Add(catchment_id)
 
+    def _is_lts_result_file(self):
+        # For pythonnet version > 3.0 it is possible to call
+        # return self._data.ResultType.Equals(ResultTypes.LTSEvents)
+        return int(self._data.ResultType) == int(ResultTypes.LTSEvents)
+
     #endregion File loading
 
     def read(self, queries=None):
```

Comment:
Not directly related to this PR, but while you have the box open. I would add a couple of examples to the read docstring. To read all data is obvious, but not how to use a query.

So one example of a query would be nice to have.

Reply:
It is a good suggestion. Also the Res1D class would benefit a docstring with all it's input parameters. I will add it.

Reply:
@ecomodeller Thanks for the review! I will merge the changes in now. I will use the principle that one review is enough if it is simple changes like this.